### PR TITLE
feat(Switch): add Switch component

### LINKED ISSUE #121: Toggle/Switch Support
Create support for Toggle/Switch component:

Include story examples demonstrating these features in a mock application scenario

Link to JIRA story:
https://patternfly.atlassian.net/browse/OSUX-347

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**What**:
Adds Switch component based on [react-bootstrap-switch ](https://www.npmjs.com/package/react-bootstrap-switch)

Storybook: https://rawgit.com/jeff-phillips-18/patternfly-react/switch-storybook/index.html

Fixes #121

### CODE REVIEW

File: src/components/Switch/Switch.stories.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { inlineTemplate } from '../../../storybook/decorators/storyTemplates';
+import { Switch } from './index';
+import { boolean, select, withKnobs } from '@storybook/addon-knobs/dist/index';
```
Comment (priley86) [inline src/components/Switch/Switch.stories.js]: ```import { withKnobs, select, boolean } from '@storybook/addon-knobs'; ```
 should work w/o the path ref...

Comment (jeff-phillips-18) [reply src/components/Switch/Switch.stories.js]: done, thanks.

File: src/components/Switch/Switch.stories.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        readonly={boolean('readonly', false)}
+        inverse={boolean('inverse', false)}
+        tristate={boolean('tristate', false)}
+      />
+    </div>
+  );
+  return inlineTemplate({
+    title: 'Switch',
+    documentationLink:
+      DOCUMENTATION_URL.PATTERNFLY_ORG_WIDGETS + '#bootstrap-switch',
```
Comment (cdcabrera) [inline src/components/Switch/Switch.stories.js]: Maybe a string literal.

File: src/components/Switch/Switch.stories.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { boolean, select, withKnobs } from '@storybook/addon-knobs';
+import { DOCUMENTATION_URL } from '../../../storybook/constants';
+
+const stories = storiesOf('Widgets', module);
+stories.addDecorator(withKnobs);
+
+stories.addWithInfo('Switch', '', () => {
+  const bsSize = select('Size', Switch.SWITCH_BS_SIZES, 'normal');
+  const onColor = select('On Color', Switch.SWITCH_COLORS, 'primary');
+  const offColor = select('Off Collor', Switch.SWITCH_COLORS, 'default');
```
Comment (cdcabrera) [inline src/components/Switch/Switch.stories.js]: Typo

Comment (jeff-phillips-18) [reply src/components/Switch/Switch.stories.js]: Fixed.

File: src/components/Switch/Switch.stories.js (hunk 2 above)
Comment (jeff-phillips-18) [reply src/components/Switch/Switch.stories.js]: Going to leave this as is, its only in this one place and consistent with other components.

**Pull request merged**